requirements.txt

```diff
@@ -0,0 +1 @@
+black==19.10b0
```

Comment:
requirements.txt exists here: https://github.com/okfn-brasil/diario-oficial/blob/main/processing/requirements.txt

Reply:
Oh, nice. I'll point to that one :)

Reply:
Actually, this requirements is for the docker container and they are likely to conflict if installing directly out of docker.
So I don't think it's good installing them in the setup. They should only be installed in the docker steps. 
@sergiomario what do you think?

Reply:
Right. I understood the conflict, I'll do the merge like this.